gnovm/tests/imports.go

```diff
@@ -636,6 +636,29 @@ func testPackageInjector(store gno.Store, pn *gno.PackageNode) {
 				m.Context = ctx
 			},
 		)
+		pn.DefineNative("TestSkipTimestamps",
+			gno.Flds( // params
+				"timestamp", gno.AnyT(), // NOTE: should be time.Duration
+			),
+			gno.Flds( // results
+			),
+			func(m *gno.Machine) {
+				arg0 := m.LastBlock().GetParams1().TV
```

Comment:
Check that arg0 is an int64 or panic with a more helpful error message? (ie. `Test... accepts time.Duration as an argument`)

Reply:
Option1. change L#641 `gno.AnyT()` to `int64` which will panic automatically if arg0 isn't type of `int64`

Option2. add some verficiation logic
```go
if arg0.T != gno.Int64Type {
  panic("TestSkipTimestamps: arg0 is not int64")
}
```

Reply:
Would it not panic if we called it with a time.Duration parameter, if the argument was of type `int64`? It's a type, not an alias, so AFAIK it should

Reply:
removed `TestSkipTimestamps` ( it has exactly same code as `internal/os_test Sleep()`